src/css/components/grid.scss

```diff
@@ -0,0 +1,9 @@
+.cg-grid {
```

Comment:
Now that I think about it, we don't use `cg` prefix for anything else. Could it just be `grid`? Or do you think it's a better interface to have the `cg` prefix?